### PR TITLE
- ported to 3.4.1 with backward compatibility

### PR DESCRIPTION
**Pull request by Eloar**
Changes allowed me to successfully install and test pytube on Python 3.4.1.

### CODE REVIEW

**Comment by josegonzalez** [comment]
I think you need to import print from futures for this.

**Comment by Eloar** [comment]
There is no advanced features used in print so according to [this](http://python3porting.com/noconv.html#supporting-the-print-function), print() should work just fine.

I tested it on 3.4.1 only, at the moment.

**Comment by josegonzalez** [comment]
That won't work on Python 2.6, in which case you need to use **future**.

**Comment by Eloar** [comment]
Is it ok now, or I missed something else?

File: README.rst [hunk 1]
```diff
@@ -95,7 +95,7 @@ Usage Example
     # You can override this by manually setting the filename.
 
     # view the auto generated filename:
-    print yt.filename
+    print(yt.filename)
```
Comment (josegonzalez) [inline README.rst:99]: Probably makes sense to add the import in this example as well.

**Comment by Eloar** [comment]
ok, added import to example

**Comment by josegonzalez** [comment]
Works for me :) Can you add the fact that we support 3.4 to the readme and setup.py as well? I can merge after that :)

**Comment by Eloar** [comment]
done

**Pull request closed without merging**